listings_app/settings.py

```diff
@@ -11,6 +11,11 @@
 """
 
 import os
+import environ
+
+ROOT_DIR = environ.Path(__file__) - 2  # two folder back to root # (listings-app/listings_app/settings.py - 2 = listings-app/)
+env = environ.Env()
+env.read_env(str(ROOT_DIR.path('.env')))
```

Comment:
I would prefer to use `env_utils` https://pypi.org/project/env_utils/ it is commonly used

```env_utils import (
    get_bool,
    get_env,
    get_int,
    get_list,
)``

This way we can set env like this: `export DATABASE_URL=something` and it works with Heroku as well as soon we deploy this. And to read it is is simply `DATABASE_URL = get_env('DATABASE_URL')`

The proposed approach seems to require a hard file & changing our ROOT_DIR which I feel is not needed. Was there a particular reasoning behind this approach?

Reply:
I was replicating a pattern used in django-cookiecutter.  I've not used env_utils am more than happy to use it. Let me read through its documentation and re-submit.